cmds/core/tail/tail.go

```diff
@@ -267,26 +267,44 @@ func run(reader *os.File, writer io.Writer, follow bool, numLines int, followDur
 		inFile *os.File
 		err    error
 	)
-	switch len(args) {
-	case 0:
-		inFile = reader
-	case 1:
-		inFile, err = os.Open(args[0])
-		if err != nil {
-			return err
-		}
-	default:
-		// TODO support multiple files
-		return fmt.Errorf("tail: can only read one file at a time")
-	}
 
 	// TODO: add support for parsing + (from beggining of the file)
 	// negative sign is the same as none
 	if numLines < 0 {
 		numLines = -1 * numLines
 	}
 	config := tailConfig{follow: follow, numLines: uint(numLines), followDuration: followDuration}
```

Comment:
since we are in the tail program, once could argue that tailConfig stutters, as reviewers sometimes say. 
Maybe just call this config?

Reply:
I realize that's not your code ;-)